src/io.rs

```diff
@@ -0,0 +1,6 @@
+//! Convienence functions for reading and writing subunit packets in different IO models
```

Comment:
Convenience

Reply:
Can you perhaps fix this in the next PR? Doesn't really seem important enough to block the entire tinhg on.